app/models/minion.rb

```diff
@@ -0,0 +1,4 @@
+# Minion represents the minions that have been registered in this application.
+class Minion < ApplicationRecord
+  default_scope { order hostname: :asc }
```

Comment:
It should be possible to override with the [unscoped](http://apidock.com/rails/ActiveRecord/Scoping/Default/ClassMethods/unscoped) method no ? Otherwise, I'd agree to remove this line altogether to be honest 😬